scale/job/configuration/json/job_config_v6.py

```diff
@@ -156,11 +156,7 @@ def __init__(self, config=None, existing=None, do_validate=False):
         if 'version' not in self._config:
             self._config['version'] = SCHEMA_VERSION
 
-        if self._config['version'] == '2.0':
-            # v2 and v6 are identical
-            self._config['version'] = SCHEMA_VERSION
```

Comment:
This was added back in because we were getting 500 errors for job details pages, the job types api, etc.  We need to either keep this or add calls to convert schemas.

Reply:
Check has been added back in.